### PR TITLE
Clusters list tests

### PR DESCRIPTION
**Pull request by ograu**
Towards #759

Tests added:

- The user is redirected to list when cluster doesn't exist
- Cluster list shows all clusters, each one with its details and resources, for the selected organization.

And refactored tests in `ClusterManagement.js`

### CODE REVIEW

File: __tests__/ClusterManagement.js [hunk 1]
```diff
@@ -208,12 +180,64 @@ details view`, async () => {
...
+  // Expect we have been redirected to the clusters list.
+  await wait(() => expect(getByTestId('clusters-list')).toBeInTheDocument());
+
+  const flashMessage = document.querySelector('#noty_layout__topRight');
+  expect(flashMessage).toContainHTML(
+    `Cluster <code>f4ke1</code> doesn't exist.`
+  );
+});
+
+it.only('Cluster list shows all clusters, each one with its details, for the selected organization', async () => {
```
Comment (axbarsan) [inline __tests__/ClusterManagement.js]: I think you forgot this here 😬 

```suggestion
it('Cluster list shows all clusters, each one with its details, for the selected organization', async () => {
```

Comment (ograu) [reply __tests__/ClusterManagement.js]: 🤦‍♂

File: src/index.html [hunk 2]
```diff
@@ -28,20 +28,20 @@
...
-        azureCapabilitiesJSON: '{"Standard_A2_v2":{"additionalProperties":{},"description":"This is some description","maxDataDiskCount":4,"memoryInMb":4294.967296,"name":"Standard_A2_v2","numberOfCores":2,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":21474.83648},"Standard_A4_v2":{"additionalProperties":{},"description":"Here is a longer description that might be too long for the field","maxDataDiskCount":8,"memoryInMb":8589.934592,"name":"Standard_A4_v2","numberOfCores":4,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":42949.67296},"Standard_A8_v2":{"additionalProperties":{},"description":"Another VM size description text","maxDataDiskCount":16,"memoryInMb":17179.869184,"name":"Standard_A8_v2","numberOfCores":8,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":85899.34592}, "Standard_D2s_v3":{"additionalProperties":{},"description":"Dsv3-series, general purpose, 160-190 ACU, premium storage supported","maxDataDiskCount":4,"memoryInMb":8589.935,"name":"Standard_D2s_v3","numberOfCores":2,"osDiskSizeInMb":1047552,"resourceDiskSizeInMb":17179.87}}',
-        happaVersion: 'development',
-      };
+  apiEndpoint: "https://api.g8s.gauss.eu-central-1.aws.gigantic.io",
```
Comment (axbarsan) [inline src/index.html]: Somehow I didn't see this during the review...

Comment (ograu) [reply src/index.html]: Deja vu!
Time to automate this.
 🙏

Comment (axbarsan) [reply src/index.html]: I fixed it in https://github.com/giantswarm/happa/pull/1177 , if it makes any difference 😂

Comment (ograu) [reply src/index.html]: yeah, reviewing it... thanks again! 🙂

**Pull request merged**